Core/src/org/sleuthkit/autopsy/centralrepository/contentviewer/DataContentViewerOtherCases.java

```diff
@@ -414,64 +422,133 @@ private AbstractFile getAbstractFileFromNode(Node node) {
             } catch (EamDbException ex) {
                 LOGGER.log(Level.SEVERE, "Error connecting to DB", ex); // NON-NLS
             }
+
+        } else {
+            try {
+                // If EamDb not enabled, get the Files default correlation type to allow Other Occurances to be enabled.   
+                if(this.file != null) {
```

Comment:
Sorry one more thing. I think you need to do the same checks from above about the MD5 being null or empty. I'm getting stack traces in the logs when I click on things that have no hash (for example, an unalloc space file)